e2e-tests/tfm/cypress/e2e/journeys/case-amendments/6. cover-end-date.spec.js

```diff
@@ -124,10 +124,7 @@ context('Amendments - Cover End Date', () => {
     amendmentsPage.amendmentCoverEndDateMonthInput().clear().focused().type(dateConstants.todayMonth);
     amendmentsPage.amendmentCoverEndDateYearInput().clear().focused().type(dateConstants.todayYear);
     amendmentsPage.continueAmendment().click();
-    cy.url().should('contain', 'check-answers');
 
-    amendmentsPage.amendmentAnswerBankRequestDate().should('contain', dateConstants.todayDay);
-    amendmentsPage.amendmentAnswerRequireApproval().should('contain', 'Yes');
-    amendmentsPage.amendmentAnswerCoverEndDate().should('contain', dateConstants.todayDay);
+    cy.url().should('contain', 'is-using-facility-end-date');
```

Comment:
fwiw we can do a more strict check 
```suggestion
    cy.url().should('eq', relative('blah/blah/is-using-facility-end-date'));
```